src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/bail/homeoffice/email/HomeOfficeBailApplicationSubmittedPersonalisation.java

```diff
@@ -0,0 +1,66 @@
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.BailCase;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.BailCaseFieldDefinition;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.ccd.field.YesOrNo;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.personalisation.BailEmailNotificationPersonalisation;
+
+@Service
+public class HomeOfficeBailApplicationSubmittedPersonalisation implements BailEmailNotificationPersonalisation {
+
+    private final String homeOfficeBailApplicationSubmittedWithLegalRepPersonalisationTemplateId;
+    private final String homeOfficeBailApplicationSubmittedWithoutLegalRepPersonalisationTemplateId;
+    private final String bailHomeOfficeEmailAddress;
+
+
+    public HomeOfficeBailApplicationSubmittedPersonalisation(
+        @NotNull(message = "homeOfficeBailApplicationSubmittedPersonalisationTemplateId cannot be null")
+        @Value("${govnotify.template.bail.submitApplication.email}") String homeOfficeBailApplicationSubmittedWithLegalRepPersonalisationTemplateId,
+        @Value("${govnotify.template.bail.submitApplicationWithoutLR.email}") String homeOfficeBailApplicationSubmittedWithoutLegalRepPersonalisationTemplateId,
+        @Value("${bailHomeOfficeEmailAddress}") String bailHomeOfficeEmailAddress
+    ) {
+        this.homeOfficeBailApplicationSubmittedWithLegalRepPersonalisationTemplateId = homeOfficeBailApplicationSubmittedWithLegalRepPersonalisationTemplateId;
+        this.homeOfficeBailApplicationSubmittedWithoutLegalRepPersonalisationTemplateId = homeOfficeBailApplicationSubmittedWithoutLegalRepPersonalisationTemplateId;
+        this.bailHomeOfficeEmailAddress = bailHomeOfficeEmailAddress;
+    }
+
+    @Override
+    public String getReferenceId(Long caseId) {
+        return caseId + "_BAIL_APPLICATION_SUBMITTED_HOME_OFFICE";
+    }
+
+    @Override
+    public String getTemplateId(BailCase bailCase) {
+        return  bailCase.read(IS_LEGALLY_REPRESENTED_FOR_FLAG, YesOrNo.class).orElse(YesOrNo.NO) == YesOrNo.YES
+            ? homeOfficeBailApplicationSubmittedWithLegalRepPersonalisationTemplateId : homeOfficeBailApplicationSubmittedWithoutLegalRepPersonalisationTemplateId;
+    }
+
+    @Override
+    public Set<String> getRecipientsList(BailCase bailCase) {
+        return Collections.singleton(bailHomeOfficeEmailAddress);
+    }
+
+    @Override
+    public Map<String, String> getPersonalisation(BailCase bailCase) {
+        requireNonNull(bailCase, "bailCase must not be null");
+
+        return ImmutableMap
+            .<String, String>builder()
+            .put("bailReferenceNumber", bailCase.read(BailCaseFieldDefinition.BAIL_REFERENCE_NUMBER, String.class).orElse(""))
+            .put("legalRepReference", bailCase.read(BailCaseFieldDefinition.LEGAL_REP_REFERENCE, String.class).orElse(""))
```

Comment:
@neha-aggarw, I believe this entry is added no matter which template is selected, and I don't think it will cause any problem since it'll just be ignored. Whereas our personalisations does the following instead, do you think this is ok?:
![image](https://user-images.githubusercontent.com/99262639/165351661-1e2df7e3-8a5b-4bc4-a021-0a1df72eb855.png)

Reply:
I think we can leave in as such as well as it will be ignored.